### DISCUSSION TITLE
[Feature] Searching on shared albums

### DISCUSSION THREAD

**Discussion by laurfb** [immich-app/immich#11474] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi. I think will be useful to be able to search on shared albums (now seems that we can only browse on shared albums/resources).
regards,
Laur

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2995
